spec/activefedora/ordered_spec.rb

```diff
@@ -62,6 +61,27 @@ class Image < ActiveFedora::Base
         expect(subject.ordered_member_proxies.to_a.length).to eq 1
       end
     end
+    describe "#=" do
+      it "can set ordered members" do
+        member = Member.new
+        member_2 = Member.new
+        subject.ordered_members << member
+        expect(subject.ordered_members).to eq [member]
+        subject.ordered_members = [member_2, member_2]
+        expect(subject.ordered_members).to eq [member_2, member_2]
+        # Removing from ordering is not the same as removing from aggregation.
+        expect(subject.members).to eq [member, member_2]
+      end
+    end
+    describe "+=" do
+      it "can append ordered members" do
```

Comment:
I have a preference for saying `it "appends ordered members"`. Using "can" here is akin to using "should".